.goreleaser.yml

```diff
@@ -120,5 +120,17 @@ release:
   # Default is `{{.Tag}}`
   name_template: "{{.ProjectName}}-v{{.Version}} {{.Env.USER}}"
 
-# brews:
-# We do not use the brew config, which is for taps, not core forumulas. 
+brews:
```

Comment:
is there any way we can test this?

Reply:
As far as I'm concerned, I am not aware of a way to explicitly test `brews` section locally. However, I use this

`goreleaser release --snapshot --clean -verbose`

Running this command with the proposed changes should give you verbose output, including a section on `homebrew tap formula`.  Speaking of which, it's called `repository` now as `brews.tap` is deprecated. To reproduce run `goreleaser check` and read https://goreleaser.com/deprecations#brewstap

**Note: I need to comment out this section:**

```
#signs:
#  - artifacts: checksum
#    args: ["--batch", "-u", "{{ .Env.GPG_FINGERPRINT }}", "--output", "${signature}", "--detach-sign", "${artifact}"]
```

As I don't have access to `Env.GPG_FINGERPRINT` and without this `gpg` complains and `goreleaser` fails locally.

Running the above command with the proposed changes in this PR, gives me the following (redacted for brevity):

```
[output shortened]

• homebrew tap formula
    • token type                                     type=github
    • calculating checksum for dist/exercism-0.0.0-SNAPSHOT-83a63ba-linux-armv6.tar.gz
    • guessing install                               install=[bin.install "exercism"]
    • calculating checksum for dist/exercism-0.0.0-SNAPSHOT-83a63ba-linux-x86_64.tar.gz
    • guessing install                               install=[bin.install "exercism"]
    • calculating checksum for dist/exercism-0.0.0-SNAPSHOT-83a63ba-darwin-x86_64.tar.gz
    • guessing install                               install=[bin.install "exercism"]
    • calculating checksum for dist/exercism-0.0.0-SNAPSHOT-83a63ba-linux-arm64.tar.gz
    • guessing install                               install=[bin.install "exercism"]
    • calculating checksum for dist/exercism-0.0.0-SNAPSHOT-83a63ba-darwin-arm64.tar.gz
    • guessing install                               install=[bin.install "exercism"]
    • writing                                        formula=dist/homebrew/Formula/exercism.rb
    • added new artifact                             name=exercism.rb type=Brew Tap path=dist/homebrew/Formula/exercism.rb

[output shortened]
```

This indicates that the brews section is being processed correctly and the actual versioned release should generate and publish a homebrew-tap recipe into a repository.